src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-detail/cephfs-detail.component.html

```diff
@@ -44,6 +44,11 @@
                        *ngIf="clientsSelect">
     </cd-cephfs-clients>
   </tab>
+  <tab heading="Performance Details">
```

Comment:
The attribute ``i18n-heading`` is missing to translate the text.

Reply:
ack